app/post/directives/post-detail-actions-directive.js

```diff
@@ -0,0 +1,50 @@
+module.exports = [
+    '$rootScope',
+    '$translate',
+    'PostEndpoint',
+    'Notify',
+    '$location',
+function (
+    $rootScope,
+    $translate,
+    PostEndpoint,
+    Notify,
+    $location
+) {
+    return {
+        restrict: 'E',
+        replace: true,
+        scope: {
+            post: '='
+        },
+        templateUrl: 'templates/posts/post-detail-actions.html',
+        link: function ($scope) {
+            $scope.showDropDown = false;
+            $scope.toggleDropDown = function () {
+                $scope.showDropDown = !$scope.showDropDown;
+            };
+
+            $scope.addToCollection = function (post) {
+                $rootScope.$emit('collectionToggle:show', [post]);
+            };
+
+            $scope.delete = function (post) {
```

Comment:
Can we share this code with the delete function in the listing?
Probably move it to something a PostActionsService. The confirm + notify part of the pattern is repeated, only the redirect is different.